paasta_tools/vitesscluster_tools.py

```diff
@@ -775,16 +782,50 @@ def get_global_lock_server(self) -> Dict[str, Dict[str, str]]:
             }
         }
 
+    def get_kubernetes_volumes(self) -> List[KubernetesVolume]:
+        system_volumes = load_system_paasta_config().get_volumes()
+        docker_volumes = self.get_volumes(
```

Comment:
we should hardcode passing in `uses_bulkdata_default=False` to all the self.get_volumes() calls so that we skip adding bulkdata mounts (that y'all don't need)

Reply:
Looking again at it, the get_volumes [method](https://github.com/Yelp/paasta/blob/master/paasta_tools/utils.py#L1003) of `InstanceConfig` has the uses_bulkdata_default argument but not SystemPaastaConfig [one](https://github.com/Yelp/paasta/blob/master/paasta_tools/utils.py#L2210)